container-engine-lib/lib/backend_impls/kubernetes/kubernetes_kurtosis_backend/user_services_functions/stop_user_services.go

```diff
@@ -46,22 +45,6 @@ func StopUserServices(
 			}
 		}
 
-		kubernetesService := resources.Service
```

Comment:
Keep the k8s service untouched when the user service is stopped so we can restart it easily by attaching a new pod.  This change does not impact the remove service flow.

Reply:
Oh interesting, so when you say it doesn't affect the remove service behaviour, you mean you moved this piece of logic to the remove service function? I.e. just want to make sure that when a "kurtosise user service is removed", we don't leave any dangling resource in kubernetes

Reply:
The code I removed set the k8s service selector to nil to not associate itself to a pod anymore.  The k8s service stayed behind after removing the user service.  I did not change that behavior.  The k8s services are cleaned up only when the k8s namespace is deleted during enclave deletion.  I now keep the selector set to be able to associate a new pod.  This change is transparent to the user since the pod is still removed.